.travis.yml

```diff
@@ -4,4 +4,4 @@ rvm:
 before_script:
   - gem install awesome_bot
 script:
-  - awesome_bot README.md --allow-redirect -w agilion.com,brighterlink.io,dubber.net,therealreal.com,marketfactory.com
+  - awesome_bot README.md --allow-redirect -w agilion.com,brighterlink.io,dubber.net,therealreal.com,marketfactory.com,www.high-mobility.com
```

Comment:
I would rather you not add yourself to the whitelist, this is more for exceptions where the awesome_bot doesn't handle redirects correctly.